channels/applicationinsights-channel-js/Tests/Sender.tests.ts

```diff
@@ -445,7 +445,7 @@ export class SenderTests extends TestClass {
 
                 // Assert baseData
                 QUnit.assert.ok(baseData.name);
-                QUnit.assert.equal("Some name given", baseData.data);
+                QUnit.assert.equal("some data", baseData.data);
```

Comment:
Hi @MSNev , I noticed you committed last time on this file. Do you know why we assign "name" value to "data" value when "data" value has content?